src/molecules/Form/Field/Checkbox.jsx

```diff
@@ -13,7 +13,7 @@ export const FormFieldCheckbox = (props) => {
 
   return (
     <FormGroup className={`marLeft ${className} ${edited ? 'success-highlight' : ''}`} check {...rest}>
-      <Input type='checkbox' onChange={handleChange} checked={value} {...inputProps} />
+      <Input type='checkbox' onChange={handleChange} checked={value} clearable={clearable.toString()} />
```

Comment:
@bthornton505 - Why use `clearable.toString()` here? My understanding is that it's passed as boolean value (or undefined), which should translate ok to the base `<Input>` element.

If it was failing as `undefined`, you can use double "bang" to ensure a boolean true/false value is always passed 

`clearable={!!clearable}`

Reply:
Here is the console warning that was being thrown: 
<img width="610" alt="Screen Shot 2020-08-24 at 9 57 10 PM" src="https://user-images.githubusercontent.com/38757518/91121450-0a70e180-e655-11ea-9849-7cd81f0de261.png">

This explains why I'm forcing the clearable prop to a string when passed into the Input component. Please let me know if this solution makes more sense after seeing the warning message.

Reply:
:+1: